app/views/admin/truework_income_validations/_truework_income_validation.html.haml

```diff
@@ -0,0 +1 @@
+= render partial: 'admin/validations/validation', locals: {validation: truework_income_validation}

```

Comment:
With the file renamed, I also have to change this code to:
```
= render partial: 'admin/validations/validation', locals: {validation: true_work_income_validation}
```
to get it to render without error.